### DISCUSSION TITLE
Image with external link

### DISCUSSION THREAD

**Discussion by markusmoenig** [McShelby/hugo-theme-relearn#575] [Q&A]
Hi,

how can I do an image with an external link ?

```markdown
[![Image](./myimage.png)](https://test.com)
```

opens the image but not the external site ? thanks!

**Comment by McShelby** [comment]
The lightbox gets in your way, so do:

````
[![Image](./myimage.png?lightbox=false)](https://test.com)
````

Anyways, the CSS style in this cause could be improved.

See #576

**Reply by markusmoenig** [reply]
Ok, great, thanks, works fine!
